core/trino-main/src/main/java/io/trino/sql/planner/HashBucketFunction.java

```diff
@@ -0,0 +1,50 @@
+/*
```

Comment:
`Make HashBucketFunction public and extract` -> `Make HashBucketFunction public and extract as top level class`